apps/src/sharedComponents/SystemDialog/ConfirmCancelFooter.jsx

```diff
@@ -15,21 +15,20 @@ export default class ConfirmCancelFooter extends React.Component {
   static propTypes = {
     onConfirm: PropTypes.func.isRequired,
     onCancel: PropTypes.func.isRequired,
-    confirmText: Button.propTypes.text,
-    cancelText: Button.propTypes.text,
-    confirmColor: Button.propTypes.color.isRequired,
-    cancelColor: Button.propTypes.color.isRequired,
+    confirmText: PropTypes.string,
+    cancelText: PropTypes.string,
+    confirmColor: PropTypes.oneOf(Object.values(buttonColors)).isRequired,
+    cancelColor: PropTypes.oneOf(Object.values(buttonColors)).isRequired,
```

Comment:
These props are defined in the `defaultProps`. Do they need to be marked as `isRequired`?